src/components/JobsPage/Jobs.js

```diff
@@ -84,12 +84,14 @@ const Jobs = ({
         const newJobs = jobs.map(job => {
           if (match.params.pageTab === SCHEDULE_TAB) {
             return {
+              createdTime: new Date(job.creation_time),
               func: job.scheduled_object.task.spec.function,
               name: job.name,
-              type: job.kind === 'pipeline' ? 'workflow' : job.kind,
-              createdTime: new Date(job.creation_time),
               nextRun: new Date(job.next_run_time),
-              scheduled_object: job.scheduled_object
+              scheduled_object: job.scheduled_object,
+              start_time: new Date(job.last_run.status.start_time),
```

Comment:
In case for any reason backend didn't provide the last run object in `last_run` property, it will be `null` instead, so in order not to crash the UI in this case, we better use the optional chaining operator (`?.` ):
```suggestion
              start_time: new Date(job.last_run?.status.start_time),
```
And in the table cell if this detail doesn't exist we can display the text: N/A